assets/js/theme/common/carousel.js

```diff
@@ -1,5 +1,5 @@
 import $ from 'jquery';
-import 'slick-carousel/slick/slick';
+import 'slick-carousel/slick/slick.min';
```

Comment:
@sacr3dc0w - this is great!  can we put them in the `webpack.conf.js` resolve instead:

```
    resolve: {
        alias: {
            html5-history-api: path.resolve(__dirname, 'node_modules/html5-history-api/history.min',
            jquery: path.resolve(__dirname, 'node_modules/jquery/dist/jquery.min.js'),
            jstree: path.resolve(__dirname, 'node_modules/jstree/dist/jstree.min.js'),
            slick-carousel: path.resolve(__dirname, 'node_modules/slick-carousel/slick/slick.min'),
        },
    },
```

that way the imports can simply be:

```
import 'slick-carousel';
...
import 'html5-history-api'
```

seeing as this was already merged, feel free to open another PR